### DISCUSSION TITLE
2007/06/19/%E5%AE%9A%E6%97%B6%E7%BB%93%E6%9E%84

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#166] [Announcements]
# 2007/06/19/%E5%AE%9A%E6%97%B6%E7%BB%93%E6%9E%84

定时结构是从 LabVIEW 7.1 开始出现的。一眼就能看出来，它在外观与其它结构的风格完全不同。倒是和 LabVIEW 7 力推的 Express VI，风格一致。打开定时结构的函数面板（图1），最上面两个分别是定时循环，和定时顺序结构。下面的是与控制时间结构相关的的一些VI。

https://blog.qizhen.xyz/2007/06/19/%E5%AE%9A%E6%97%B6%E7%BB%93%E6%9E%84

**Comment by ruanqizhen** [comment]
之前博客下的留言：

刘伟印
2008年08月20日 10:27 编辑
我没用过定时结构，想不到wait的精确度那么差啊

回复
geon
2007年12月17日 01:01 编辑
这条信息很重要
 
我原来测试过wait的分辨率为1ms
可是精度和最小wait时间都为2ms左右

原来以为在windows系统的PC下
wait和定时结构二者精度差不多呢
